src/main/java/knightminer/simplytea/core/config/Teapot.java

```diff
@@ -2,11 +2,15 @@
 
 import net.minecraftforge.common.ForgeConfigSpec;
 import net.minecraftforge.common.ForgeConfigSpec.BooleanValue;
+import net.minecraftforge.common.ForgeConfigSpec.IntValue;
+import net.minecraftforge.fluids.FluidType;
 
 public class Teapot {
   private BooleanValue infinite_water;
   private BooleanValue fill_from_cauldron;
+  private BooleanValue fill_from_fluid_tank;
```

Comment:
I think this was forgotten during cleanup.

Reply:
removed.